Shared/Samples/Augment reality to show hidden infrastructure/AugmentRealityToShowHiddenInfrastructureView.ARSceneView.swift

```diff
@@ -0,0 +1,177 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+extension AugmentRealityToShowHiddenInfrastructureView {
+    /// A world scale scene view displaying pipe graphics from a given model.
+    struct ARPipesSceneView: View {
+        /// The view model for scene view in the sample.
+        @ObservedObject var model: SceneModel
+        
+        /// A Boolean value indicating whether the shadow graphics are visible.
+        @State private var shadowsAreVisible = true
+        
+        /// A Boolean value indicating whether the leader line graphics are visible.
+        @State private var leadersAreVisible = true
+        
+        var body: some View {
+            WorldScaleSceneView { _ in
+                SceneView(scene: model.scene, graphicsOverlays: [
+                    model.pipeGraphicsOverlay,
+                    model.shadowGraphicsOverlay,
+                    model.leaderGraphicsOverlay
+                ])
+            }
+            .calibrationButtonAlignment(.bottomLeading)
+            .onCalibrationViewVisibilityChanged { isPresented in
+                model.scene.baseSurface.opacity = isPresented ? 0.6 : 0
```

Comment:
```suggestion
            .onCalibratingChanged { newCalibrating in
                model.scene.baseSurface.opacity = newCalibrating ? 0.6 : 0
```
The modifier name was updated.